i18n4k-core/build.gradle.kts

```diff
@@ -33,9 +33,8 @@ kotlin {
     }
     // #####  native targets...
     // # out commented targets are not supported by a used library
+    ios()
     iosArm32()
-    iosArm64()
-    iosX64()
```

Comment:
As `ios()` is just a short cut for `iosArm64` and `iosX64`. Not for all ios targets.

Therefor the ios source set can than be creates explicitly with an additional "apple" source set. (See other comments)